ydb/_grpc/grpcwrapper/common_utils.py

```diff
@@ -112,15 +113,16 @@ def __next__(self):
 
 
 class SyncIteratorToAsyncIterator:
-    def __init__(self, sync_iterator: Iterator):
+    def __init__(self, sync_iterator: Iterator, executor: concurrent.futures.Executor):
         self._sync_iterator = sync_iterator
+        self._executor = executor
 
     def __aiter__(self):
         return self
 
     async def __anext__(self):
         try:
-            res = await to_thread(self._sync_iterator.__next__)
+            res = await to_thread(self._sync_iterator.__next__, executor=self._executor)
             return res
         except StopAsyncIteration:
             raise StopIteration()
```

Comment:
why we are rasing `StopIteration` instead of `StopAsyncIteration` here?
https://docs.python.org/3/library/functions.html#anext

Reply:
Fixed in https://github.com/ydb-platform/ydb-python-sdk/pull/292